mlflow/pytorch/_pytorch_autolog.py

```diff
@@ -0,0 +1,221 @@
+import gorilla
+import logging
+import mlflow.pytorch
+import os
+import pytorch_lightning as pl
+import shutil
+import tempfile
+from pytorch_lightning.core.memory import ModelSummary
+from pytorch_lightning.utilities import rank_zero_only
+from mlflow.utils.autologging_utils import try_mlflow_log, wrap_patch
+
+logging.basicConfig(level=logging.ERROR)
+
+every_n_epoch = 1
```

Comment:
I found the tensorflow does the same thing (defining a constant and use it with `global` in the autolog function). @smurching Do you happen to know why we need to do this?

https://github.com/mlflow/mlflow/blob/master/mlflow/tensorflow.py#L50

Reply:
ah I got it. `_LOG_EVERY_N_STEPS` is used in `_log_event`.

https://github.com/mlflow/mlflow/blob/master/mlflow/tensorflow.py#L601